travis_before_install.sh

```diff
@@ -29,6 +29,7 @@ if [ "$TRACK" = "ansible" ]; then
     sudo apt-get -y --purge remove postgresql libpq-dev libpq5 postgresql-client-common postgresql-common
     sudo rm -rf /var/lib/postgresql
     sudo apt-get update -qq
+    ansible-galaxy install -r requirements_roles.yml -p roles
```

Comment:
Ah ! I see. No, the idea with the metavisitor is to provide all roles included, avoiding to fetch independent roles a earliest stages.
Thus, I start to understand the issue with this PR: The Tag galaxykickstart should include the fetched roles. I don't remember how I did that the first time, but I will probably find it in GKS repo (so do you).